### ISSUE TITLE
Invoker bug

### ISSUE THREAD

**Issue by Wumbo31** [ValveSoftware/Dota2-Gameplay#12555]
### Ability name

EMP, ColdSnap

### Description

No mana gain from EMP
ColdSnap does not work on medusa

### Example Match ID (and possibly Timestamp)

No match id, found using test mode

**Comment by Wumbo31** [comment]
Found using test mode.

**Comment by Wumbo31** [comment]
Found using demo mode.

**Comment by Wumbo31** [comment]
> did cold snap work on musa shield prior to 7.34d? i am confused i think it did but someone said it hasn't worked since the medusa rework.

no, just an issue i wanted to address while talking about the new one

**Comment by TobyFlendersonFromHR** [comment]
big, if true

**Comment by Avarise831** [comment]
#12562

**Comment by dweksa** [comment]
Its fixed now, so now wex invoker sucks a bit nonetheless with the constant nerfs. Give him a break!!

**Comment by irismus** [comment]
https://github.com/ValveSoftware/Dota2-Gameplay/issues/12562#issuecomment-1751219906

Fixed. Please close the issue, thank you!
